app/views/foreman/unattended/kickstart-katello.erb

```diff
@@ -22,7 +22,11 @@ oses:
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
 install
+<% if host_param('kickstart_liveimg') %>
+liveimg --url=<%= host_param('kickstart_liveimg') %>
+<% else %>
 <%= @mediapath %>
+<% end %>
```

Comment:
How does this param get set?  I don't see it referenced in foreman, or in https://github.com/theforeman/community-templates

Reply:
under the `host parameters` you define it at this time.

```
name = kickstart_liveimg
value = ..... squashfs.img
```

Reply:
any thoughts on adding support for this in the community templates?  These templates may be going away fairly soon.

Reply:
@jlsherrill it is also there: https://github.com/theforeman/community-templates/pull/455 :)